energyplus_regressions/tests/diffs/test_table_diff.py

```diff
@@ -292,7 +312,7 @@ def test_weird_unicode_issue(self):
         self.assertEqual(155, response[1])  # count_of_tables
         self.assertEqual(334, response[2])  # big diffs
         self.assertEqual(67, response[3])  # small diffs
-        self.assertEqual(3763, response[4])  # equals
+        self.assertEqual(3756, response[4])  # equals
```

Comment:
Since we aren't checking the input fields count/echo table anymore, these values had to be reduced by that many cells.  All good.